modules/nw-enabling-hsts.adoc

```diff
@@ -0,0 +1,62 @@
+// Module filename: nw-enabling-hsts.adoc
+// Module included in the following assemblies:
+// * networking/configuring-routing.adoc
+
+[id='nw-enabling-hsts-{context}']
+= Enabling HTTP Strict Transport Security
+
+HTTP Strict Transport Security (HSTS) policy is a security enhancement, which
+ensures that only HTTPS traffic is allowed on the host. Any HTTP requests are
+dropped by default. This is useful for ensuring secure interactions with
+websites, or to offer a secure application for the user's benefit.
+
+When HSTS is enabled, HSTS adds a Strict Transport Security header to HTTPS
+responses from the site. You can use the `insecureEdgeTerminationPolicy` value
+in a route to redirect to send HTTP to HTTPS. However, when HSTS is enabled, the
+client changes all requests from the HTTP URL to HTTPS before the request is
+sent, eliminating the need for a redirect. This is not required to be supported
+by the client, and can be disabled by setting `max-age=0`.
+
+[IMPORTANT]
+====
+HSTS works only with secure routes (either edge terminated or re-encrypt). The
+configuration is ineffective on HTTP or passthrough routes.
+====
+
+.Procedure
+* To enable HSTS on a route, add the `haproxy.router.openshift.io/hsts_header`
+value to the edge terminated or re-encrypt route:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Route
+metadata:
+  annotations:
+    haproxy.router.openshift.io/hsts_header: max-age=31536000;includeSubDomains;preload <1> <2> <3>
+----
++
+[IMPORTANT]
+====
+Ensure there are no spaces and no other values in the parameters in the
+`haproxy.router.openshift.io/hsts_header` value.
+====
++
+<1> `max-age` is the only required parameter.
+It measures the length of time, in seconds, that the
+HSTS policy is in effect. The client updates `max-age` whenever a response
+with a HSTS header is received from the host. When `max-age` times out, the
+client discards the policy.
++
+<2> `includeSubDomains` is optional. When included, it tells the client
+that all subdomains
+of the host are to be treated the same as the host.
++
+<3> `preload` is optional: When `max-age` is greater than 0, then including
```

Comment:
I would change ":" to "." to be consistent with the format used in the second callout.